modules/persistent-storage-csi-google-cloud-file-create-sc.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-google-cloud-file-create-sc_{context}"]
+= Creating a storage class for GCP Filestore Storage
+
+After installing the operator, you should create a storage class for dynamic provisioning of Google Compute Platform (GCP) Filestore volumes. 
+
+.Prerequisites
+* You are logged in to the running {product-title} cluster.
+
+.Procedure
+To create a storage class:
+
+. Create a storage class using the following example YAML file:
++
+[source,yaml]
+.Example YAML file
+--
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: filestore-csi
+provisioner: filestore.csi.storage.gke.io
+parameters:
+  network: network-name <1>
+allowVolumeExpansion: true
+volumeBindingMode: WaitForFirstConsumer
+--
+<1> Specify the name of the network where Filestore instances should be created in. 
+
+. Specify the name of the network where Filestore instances should be created in.
++
+It is recommended to specify the network that the Filestore instances should be created in. If no network is specified, the Container Storage Interface (CSI) driver tries to create the instances in the default network of the project.
+On IPI installations, the network name is typically the cluster name with the suffix "-network". However, on UPI installations, the network name can be any value chosen by the user.
++
+You can find out the network name by inspecting the `MachineSets` objects with the following command: 
++
+[source, command]
+----
+$ oc -n openshift-machine-api get machinesets -o yaml | grep "network:"
+            - network: gcp-filestore-network
+(...)
+----
+In this example, the network name in this cluster is "gcp-filestore-network".
```

Comment:
I suggest replacing `network` everywhere in this page.

Reply:
Done.